framework/quota/src/main/java/org/apache/cloudstack/quota/QuotaManagerImpl.java

```diff
@@ -327,6 +327,13 @@ protected List<QuotaUsageVO> createQuotaUsagesAccordingToQuotaTariffs(AccountVO
                     continue;
                 }
 
+                if (!QuotaConfig.QuotaAccountEnabled.valueIn(account.getAccountId())) {
```

Comment:
```suggestion
                if (!Boolean.TRUE.equals(QuotaConfig.QuotaAccountEnabled.valueIn(account.getAccountId()))) {
```

Reply:
also join this conditional with the preceding one. The code blocks are the same.